### ISSUE TITLE
So is this code actually working?

### ISSUE THREAD

**Issue by mario98** [yunjey/mnist-svhn-transfer#5]
So is this code actually working? Readme says "(should be re-uploaded)", What does that mean? Some people here report that this code is just memorizing training examples but does not generalize. Is this true?

Even worse than having no code is having code that is pretending to work but will just waste your time and then you have to implement it from scratch anyway...

**Comment by shashank3959** [comment]
@mario98 
Hey, were you able to get this code working?

**Comment by YeolJ00** [comment]
https://github.com/YeolJ00/mnist-svhn-transfer
I've edited the code to work and changed some functions to meet recent updates
I wanted to make a branch or a pull request in this repository but wasn't able to figure out how.

**Comment by EvaFlower** [comment]
@YeolJ00 Hi there, would you please point out the problem in this code and the change you made?

**Comment by YeolJ00** [comment]
@EvaFlower Putting aside the deprecated functions,
data_loader.py had a single tranform format in get_loader() that normalizes a 3 channel image.
MNIST data is not a 3 channel image thus requires a seperate transform format
It would be nice if you would double check it for me

**Comment by EvaFlower** [comment]
@YeolJ00 Yeah, you are right. But it did not raise exceptional information when running the code. So maybe it has no influence.

**Comment by YeolJ00** [comment]
@EvaFlower What do you mean by exceptional information? In what sense do you mean by it has no influence?

**Comment by EvaFlower** [comment]
@YeolJ00 I mean that I can run the code successfully without changing the transform for MNIST. So maybe here it just used the transform in the first channel and ignored others.

**Comment by YeolJ00** [comment]
@EvaFlower Well good for you. I wasn't able to run the code without changing that part. I wonder what the difference was.

**Comment by zxkplus** [comment]
I get a reasonable result, when set "--use_reconst_loss True" and "--use_labels True"。 the MINST's generator result is better than svhn's
